test/verify/check-host-switching

```diff
@@ -389,6 +395,23 @@ class TestHostSwitching(MachineCase, HostSwitcherHelpers):
         b.wait_not_present(".nav-item span[data-for='/@10.111.113.3']")
         b.wait_text(".nav-item span[data-for='/@10.111.113.2']", "admin @machine2")
 
+    def testNoAutoconnect(self):
+        b = self.browser
+        m2 = self.machines["machine2"]
+
+        self.login_and_go(None)
+
+        # And and connect to a second machine
+        b.click("#hosts-sel button")
+        self.add_machine(b, "10.111.113.2")
+        b.click("a[href='/@10.111.113.2']")
+        b.wait_visible("iframe.container-frame[name='cockpit1:10.111.113.2/system']")
+        self.assertIn("admin", m2.execute("loginctl"))
+        b.click("#hosts-sel button")
+        b.click("a[href='/@localhost']")
+        b.relogin()
+        time.sleep(60)
```

Comment:
That's certainly a long time, but I see that we want to make really sure. :grin: But the test is destructive, so it doesn't  hold up the nd ones.